DOCS: Add extensions documentation (#4850)
ansys/pyaedt: doc/source/User_guide/pyaedt_extensions_doc/project.rst

Import Nastran
==============

> Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it before import it.

Reviewer: ```suggestion
Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it prior to import.
```

Reviewer 2: You can import a Nastran or STL file in any 3D modeler application. You can also preview ....

The Google dev doc style guide says to use "app" rather than application, but I'm OK with whatever as long as use the term consistently throughout the doc.

> User can access the extension from the icon created in the Automation tab through the extension manager.

Reviewer: ```suggestion
The extension can be accessed from the icon created in the Automation tab through the extension manager.
```

Reviewer 2: You can access the extension from the icon created on the **Automation** tab using the Extension Manager.

Use bold for UI elements. Capitalize "Extension Manager" as its a proper noun of an Ansys component.

> The following image is the extension user interface:

Reviewer: ```suggestion
The following image shows the extension user interface:
```

.. image:: import_nastran_ui.png
  :width: 800
  :alt: Import nastran UI

> User can launch the user interface from the terminal:

Reviewer: ```suggestion
The user interface can also be launched from the terminal:
```

Reviewer 2: You can also launch the extension user interface from the terminal:

.. code::

   SET PYAEDT_SCRIPT_PORT=50051
   SET PYAEDT_SCRIPT_VERSION=2024.1
   python.exe path/to/pyaedt/workflows/project/import_nastran.py

> The available arguments are: file_path, planar, lightweight, and decimate. These arguments can be obtained from the help:

Reviewer: ```suggestion
The available arguments are: ``file_path``, ``planar``, ``lightweight``, and ``decimate``. These arguments can be obtained from the help:
```

Reviewer 2: You can obtain these arguments from the help with this command:

.. code::

   python.exe path/to/pyaedt/workflows/project/import_nastran.py --help

> For example, you can pass the input file as an argument, and in this case the user interface is not launched:

Reviewer 2: ```suggestion
This code shows how to pass the input file as an argument, which doesn't launch the user interface:
```

.. code::

   export PYAEDT_SCRIPT_PORT=50051
   export PYAEDT_SCRIPT_VERSION=2024.1
   python.exe path/to/pyaedt/workflows/project/import_nastran.py --file_path="my_file.stl"

> Fianlly, the extension can be used in a python script without the user interface with the following code:

Reviewer: ```suggestion
Finally, the extension can be used in a Python script from the command prompt with the following code:
```

Reviewer 2: Rewrite in active voice: 
Finally, this code shows how you can run the extension directly from a Python script:

> Fianlly, the extension can be used in a python script without the user interface with the following code:

Reviewer 2: ```suggestion
Finally, this code shows how to use the extension in a Python script without the user interface:
```
